NOTICE.md

```diff
@@ -153,6 +153,12 @@ natefinch/lumberjack (2.0.0)
 * Project: https://github.com/natefinch/lumberjack
 * Source:  https://github.com/natefinch/lumberjack/releases/tag/v2.0.0
 
+golang.org/x/sync (v0.0.0-20210220032951-036812b2e83c)
```

Comment:
There is no strict rule but it'll be more readable to keep the current order. I would suggest to move golang.org/x/sync between golang.org/x/net and golang.org/x/sys.